packages/netlify-cms-core/src/actions/config.js

```diff
@@ -31,6 +31,29 @@ const setDefaultPublicFolder = map => {
   return map;
 };
 
+const setSnakeCaseConfig = field => {
+  // List of options that are still using camelCase
+  const widgetKeyMap = new Map([
```

Comment:
Since we are mapping from camel case to snake case should we use:
```js
const map = {
  dateFormat: 'date_format'
  ...
}
```

Then we could use:
```js
Object.entries(map).forEach(([camel, snake]) => {
   if (field.has(camel)) {
      field = field.set(snake, field.get(camel));
    }
})
```

Reply:
Agree, that looks much clearer than my approach. Thank you!